e2e/cypress/integration/channel/archived_channels_spec.js

```diff
@@ -55,5 +67,204 @@ describe('Leave an archived channel', () => {
+        cy.createArchivedChannel(
+            {
+                name: 'archived-',
+                type: 'P',
+                teamId: testTeam.id,
+                teamName: testTeam.name,
+            },
+            [`some text message ${getRandomId()}`],
+        ).then(() => {
+            cy.visit(`/${testTeam.name}/channels/off-topic`);
+
+            // Select CTRL/⌘+k) to open the channel switcher
+            cy.typeCmdOrCtrl().type('K', {release: true});
+
+            // Start typing the name of a private channel on this team that has been archived which the test user belongs to
+            cy.get('#quickSwitchInput').type('archived-');
+            cy.wait(TIMEOUTS.HALF_SEC);
+
+            // * Suggestion list should be visible
+            cy.get('#suggestionList').should('be.visible');
+
+            // * there should be public channels as well
+            cy.get('#suggestionList').find('.icon-globe').should('be.visible');
+        });
+    });
+    it('MM-T1675 CTRL/CMD+K list private archived channels you are a member of', () => {
+        cy.visit(`/${testTeam.name}/channels/off-topic`);
+
+        // Select CTRL/⌘+k) to open the channel switcher
+        cy.typeCmdOrCtrl().type('K', {release: true});
+
+        // Start typing the name of a private channel on this team that has been archived which the test user belongs to
+        cy.get('#quickSwitchInput').type('archived-');
+        cy.wait(TIMEOUTS.HALF_SEC);
+
+        // * Suggestion list should be visible
+        cy.get('#suggestionList').should('be.visible');
+
+        // * there should be public channels as well
+        cy.get('#suggestionList').find('.icon-archive-outline').should('be.visible');
+    });
+    it('MM-T1676 CTRL/CMD+K does not show private archived channels you are not a member of', () => {
+        const otherChannelName = 'archived-not-mine';
+
+        // # As another user, create or locate a private channel that the test user is not a member of and archive the channel
+        cy.apiLogout();
+        cy.apiLogin(otherUser);
+        cy.createArchivedChannel(
+            {
+                name: otherChannelName,
+                type: 'P',
+                teamId: testTeam.id,
+                teamName: testTeam.name,
+            },
+            [`some text message ${getRandomId()}`],
+        );
+
+        // # As the test user, select CTRL/CMD+K (or ⌘+k) to open the channel switcher
+        cy.apiLogout();
+        cy.apiLogin(testUser);
+        cy.visit(`/${testTeam.name}/channels/off-topic`);
+        cy.typeCmdOrCtrl().type('K', {release: true});
+
+        // # Start typing the name of a private channel located above
+        cy.get('#quickSwitchInput').type('archived-');
+        cy.wait(TIMEOUTS.HALF_SEC);
+
+        // * Private archived channels you are not a member above are not available on channel switcher
+        cy.contains('#suggestionList', 'archived-not-mine').should('not.exist');
```

Comment:
Suggested change:

`cy.contains('#suggestionList', otherChannelName).should('not.exist');`